Authorization.md

```diff
@@ -0,0 +1,18 @@
+# Authorization
```

Comment:
If you wish to move this content; provide a redirect from this url address to the new location.